boa/src/builtins/property/mod.rs

```diff
@@ -53,71 +54,136 @@ pub struct Property {
 
 impl Property {
     /// Checks if the provided Value can be used as a property key.
+    #[inline]
     pub fn is_property_key(value: &Value) -> bool {
-        value.is_string() || value.is_symbol() // Uncomment this when we are handeling symbols.
+        value.is_string() || value.is_symbol()
     }
 
     /// Make a new property with the given value
     /// The difference between New and Default:
     ///
     /// New: zeros everything to make an empty object
     /// Default: Defaults according to the spec
+    #[inline]
     pub fn new() -> Self {
```

Comment:
It seems this `new()` function is basically a default, right? We can derive Default and just not implement `new()` (or make `new()` just an alias).